plugin-sysstat/translations/sysstat_cs.desktop

```diff
@@ -0,0 +1,2 @@
+Name[cs]=Statistiky o systému
+Comment[cs]=Zásuvný modul pro zobrazování statistiky o systému
```

Comment:
How about completely remove the `Zásuvný modul` and just leave it as `Zobrazení statistik o systému`